<!-- For backwards compatibility with the old driver files
                Will be DELETED in the future              -->

<launch>
  <group ns="leap_motion">
      <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" 
	    output="screen"/>
  </group>
</launch>
